### DISCUSSION TITLE
Peripherals registers view

### DISCUSSION THREAD

**Discussion by ghost** [xpack/vscode-xpack-extension-ts#44] [Q&A] [answered]
Hi all,

Is it possible to get the peripherals registers values from a MCU based on a RISC-V architecture?
I have a SVD file, but I don't know use it with VS Code xPack extension.

In advance thank you

Julien

**Comment by ilg-ul** [comment, marked as answer]
To use the SVDs, you need to somehow configure your debug sessions.

The VS Code xPack extension automates builds and dependencies, for the debug sessions you need to study the debug extensions.
